### DISCUSSION TITLE
Picture

### DISCUSSION THREAD

**Discussion by marek94901** [lukevella/rallly#664] [Ideas]
Hello, lately we have been using Rally quite often. I thought of these new features, improvements:

- for the authorized user to give the possibility to insert his photo
- at the beginning of creating the poll, I would like to give some picture of the given event.
- location of the meeting, perhaps an unnecessary function, but how about giving two options in the settings: a) the address of the office, for example, b) leave an empty field for completion

**Comment by lukevella** [comment]
Hey @marek94901, I'm not sure I understand your last point about the location field. Currently it's an optional field that doesn't have any special function but in the future there will be support for integrations with maps and web conferencing tools like zoom. Is that what you're referring to?

**Comment by marek94901** [comment]
The bottom line is that there would be a choice between the default address and the address that the participant can choose.
